### PR TITLE
Language Override space

### PR DESCRIPTION
**Pull request by brianteeman**
When you have a language override that is in both locations then it is displayed as site /administrator

This extra space is added because the markup is on multiple lines.

### To Test
Go to language overrides and select a language-admin pair
Create a new override and check the box for both locations
Save and check the list of overrides

### Before
image

### After
image

### CODE REVIEW

**Comment by brianteeman** [comment]
@HLeithner my reading of the drone error is that its not possible to resolve the space issue that the pr is intended to do without breaking the cs rules. Your advice please

**Comment by richard67** [comment]
> @HLeithner my reading of the drone error is that its not possible to resolve the space issue that the pr is intended to do without breaking the cs rules. Your advice please

@brianteeman Does it work without the curly brackets?

File: administrator/components/com_languages/tmpl/overrides/default.php [hunk 1]
```diff
@@ -93,12 +93,7 @@
                                     <?php echo $language; ?>
                                 </td>
                                 <td class="d-none d-md-table-cell">
-                                    <?php echo $client; ?>
-                                    <?php
-                                    if (isset($oppositeStrings[$key]) && ($oppositeStrings[$key] == $text)) {
-                                        echo '/' . $oppositeClient;
-                                    }
-                                    ?>
+                                    <?php echo $client; ?><?php if (isset($oppositeStrings[$key]) && ($oppositeStrings[$key] == $text)) {echo '/' . $oppositeClient;}?>
```
Comment (HLeithner) [inline administrator/components/com_languages/tmpl/overrides/default.php]:
```suggestion
                                    <?php
                                    echo $client;
                                    if (isset($oppositeStrings[$key]) && $oppositeStrings[$key] === $text) :
                                        echo '/' . $oppositeClient;
                                    endif;
                                    ?>
```

Comment (HLeithner) [reply administrator/components/com_languages/tmpl/overrides/default.php]: this looks a bit better and is valid for phpcs

Comment (brianteeman) [reply administrator/components/com_languages/tmpl/overrides/default.php]: thanks - i should have looked more closely

**Comment by brianteeman** [comment]
Thank @HLeithner all good now

**Comment by Kostelano** [comment]
I have tested this item :white_check_mark: successfully on b4fee5f17917ccc8409c37589042f4da48e06f31<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38626">issues.joomla.org/tracker/joomla-cms/38626</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on b4fee5f17917ccc8409c37589042f4da48e06f31<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38626">issues.joomla.org/tracker/joomla-cms/38626</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38626">issues.joomla.org/tracker/joomla-cms/38626</a>.</sub>

**Comment by laoneo** [comment]
Should we not try to avoid multi line PHP statements in layout files? If I remember correctly every PHP code in these layout files should start with <?php. To solve this issue you can make a one line statement out of it:
`<?php echo $client . (isset($oppositeStrings[$key]) && $oppositeStrings[$key] === $text ? '/' . $oppositeClient : ''); ?>`

**Comment by brianteeman** [comment]
is it in the code styles?

**Comment by brianteeman** [comment]
... and I would say it is beyond the scope of this PR as it was already on multiple lines that did not begin and end with <?

**Comment by HLeithner** [comment]
thanks

**Pull request merged**